core/irCheck.ml

```diff
@@ -473,8 +520,10 @@ struct
       ensure_effect_present_in_row (o#extract_type_equality_context ()) allowed_effects effect_name effect_typ occurrence
 
     method add_typevar_to_context id kind = {< type_var_env = Env.bind id kind type_var_env  >}
-    method remove_typevar_to_context id  = {< type_var_env = Env.unbind id type_var_env >}
+    (* dangerous, unnecessary, and ungrammatical *)
```

Comment:
Dead code?

Reply:
potentially useful for Frank... so I won't remove it now